test/test_core.py

```diff
@@ -915,6 +915,10 @@ def test_longjmp_with_and_without_exceptions(self):
     self.emcc_args.append('-fwasm-exceptions')
     for arg in ['-fwasm-exceptions', '-fno-exceptions']:
       self.do_core_test('test_longjmp.c', emcc_args=[arg])
+    # Wasm SjLj with and with new experimental EH support
+    self.require_new_wasm_eh()
```

Comment:
Doesn't adding this mean that this test now can't be run with out new eh?

If so, why not use the decorator?

Reply:
Hmm you're right; I was confused and somehow assumed we were able to run the two tests and kind of skip this part with node < 22.. Not sure why I've assumed that. 

Using the decorator means thie tests only runs on node 22. Do we run non-wasm64 core tests on node 22?

Reply:
Used the decorator: https://github.com/emscripten-core/emscripten/pull/21913/commits/6691a9242d62caec80f3d5f5be04ea83be9dd018